frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -686,6 +686,16 @@ To:
             UIManager:askForRestart()
         end,
     })
+    table.insert(self.menu_items.developer_options.sub_item_table, {
+        text = _("Anti-alias UI elements"),
+        checked_func = function()
+            return G_reader_settings:readSetting("anti_alias_ui", 1) ~= 0
+        end,
+        callback = function()
+            local old_val = G_reader_settings:readSetting("anti_alias_ui", 1)
+            G_reader_settings:saveSetting("anti_alias_ui", 1 - old_val)
```

Comment:
I guess you want 1 and 0 so it is passed as-is to the C function. While true and false feels more like what we are used to in frontend - where we can use stuff to toggle/flipNilOrTrue that are better than this ugly `1 - old_val` to switch between 1 and 0 :)
If there is only one call, you could just use there `anti_alias and 1 or 0` which is how we do it nearly everywhere, ie. in credocument.